### ISSUE TITLE
Problem updating OneLogin to OIDCv2

### ISSUE THREAD

**Issue by zackb** [pomerium/pomerium#1885]
## What happened?
OneLogin is [deprecating OIDCv1](https://developers.onelogin.com/openid-connect/api/upgrade) and when I tried to update our Pomerium deployment to the OIDCv2 urls I get an error about 
```
identity/oidc: could not retrieve groups: Unauthorized
```

## What did you expect to happen?
Updating `idp_provider_url` to the OIDCv2 version to work as it does before. 

## How'd it happen?
Using the helm chart, I changed `idp.provider.url` to `https://mycompany.onelogin.com/oidc/2`. This lead to some errors about invalid scopes, but the error message was clear. I changed the `idp.scopes` to `openid,profile,groups` (and have tried with different combinations of scopes) and then received the error

![image](https://user-images.githubusercontent.com/422599/107797902-4e919480-6d10-11eb-88c1-93548c838cc4.png)

## What's your environment like?

- Pomerium version: `0.9.5-1595441027+f9c118b`
- Server Operating System/Architecture/Cloud:
 We run on many Kubernetes cluster all over the world. The pomerium helm chart is installed on each cluster (and the share a signing key for interoperability). 
We use OneLogin with groups. Previously all profile/group information was recognized by pomerium no problem

## What's your config.yaml?

```config.yaml
rootDomain: mycompany.com
extraOpts:
jwt_claims_headers: email
policy:
- from: https://grafana.mycompany.com
  to: http://prometheus-operator-grafana.monitoring.svc.cluster.local:80
  cors_allow_preflight: true
  timeout: 30s
  pass_identity_headers: true
  allow_any_authenticated_user: true

- from: https://alertmanager.mycompany.com
  to: http://prometheus-operator-kube-p-alertmanager.monitoring.svc.cluster.local:9093
  cors_allow_preflight: true
  timeout: 30s
  pass_identity_headers: true
  allow_any_authenticated_user: true

proxy:
authenticate_service_url: https://authenticate.mycompany.com

authenticate:
idp:
    provider: onelogin
    clientID: <redacted>
    clientSecret: <redacted>
    url: https://mycompany.onelogin.com/oidc/2
    scopes: openid,email

ingress:
    enabled: false
```

## What did you see in the logs?

```logs
{"level":"debug","service":"envoy","name":"envoy","time":"2021-02-12T17:20:00Z"}
{"level":"debug","service":"envoy","name":"client","time":"2021-02-12T17:20:00Z","message":"[C8] response complete"}
{"level":"debug","service":"envoy","name":"http2","time":"2021-02-12T17:20:00Z","message":"[C53] stream closed: 0"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:00Z","message":"[C8] response complete"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:00Z","message":"[C8] destroying stream: 0 remaining"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:00Z","message":"[C53] remote close"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:00Z","message":"[C53] closing socket: 0"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:00Z","message":"[C53] SSL shutdown: rc=1"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:00Z","message":"[C53] "}
{"level":"debug","service":"envoy","name":"conn_handler","time":"2021-02-12T17:20:00Z","message":"[C53] adding to cleanup list"}
{"level":"info","service":"envoy","upstream-cluster":"pomerium-control-plane-http","method":"GET","authority":"10.52.3.67:443","path":"/ping","user-agent":"kube-probe/1.16+","referer":"","forwarded-for":"","request-id":"743231f7-7ff1-4ae9-b929-45295207607f","duration":0.822909,"size":61,"response-code":200,"response-code-details":"via_upstream","time":"2021-02-12T17:20:00Z","message":"http-request"}
{"level":"debug","service":"envoy","name":"filter","time":"2021-02-12T17:20:01Z","message":"tls inspector: new connection accepted"}
{"level":"debug","service":"envoy","name":"conn_handler","time":"2021-02-12T17:20:01Z","message":"[C54] new connection"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:01Z","message":"[C54] handshake expecting read"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:01Z","message":"[C54] handshake expecting read"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:01Z","message":"[C54] handshake complete"}
{"level":"debug","service":"envoy","name":"http2","time":"2021-02-12T17:20:01Z","message":"[C54] updating connection-level initial window size to 268435456"}
{"level":"debug","service":"envoy","name":"http","time":"2021-02-12T17:20:01Z","message":"[C54] new stream"}
{"level":"debug","service":"envoy","name":"http","time":"2021-02-12T17:20:01Z","message":"\"[C54][S18429224963929918799] request headers complete (end_stream=true):\\n\\':authority\\', \\'10.52.3.67:443\\'\\n\\':method\\', \\'GET\\'\\n\\':path\\', \\'/ping\\'\\n\\':scheme\\', \\'https\\'\\n\\'user-agent\\', \\'kube-probe/1.16+\\'\\n\\'accept-encoding\\', \\'gzip\\'\""}
{"level":"debug","service":"envoy","name":"envoy","time":"2021-02-12T17:20:01Z"}
{"level":"debug","service":"envoy","name":"http","time":"2021-02-12T17:20:01Z","message":"[C54][S18429224963929918799] request end stream"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:01Z","message":"coroutine finished"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:01Z","message":"coroutine finished"}
{"level":"debug","service":"envoy","name":"router","time":"2021-02-12T17:20:01Z","message":"\"[C54][S18429224963929918799] cluster \\'pomerium-control-plane-http\\' match for URL \\'/ping\\'\""}
{"level":"debug","service":"envoy","name":"router","time":"2021-02-12T17:20:01Z","message":"\"[C54][S18429224963929918799] router decoding headers:\\n\\':authority\\', \\'10.52.3.67:443\\'\\n\\':method\\', \\'GET\\'\\n\\':path\\', \\'/ping\\'\\n\\':scheme\\', \\'http\\'\\n\\'user-agent\\', \\'kube-probe/1.16+\\'\\n\\'accept-encoding\\', \\'gzip\\'\\n\\'x-forwarded-proto\\', \\'https\\'\\n\\'x-request-id\\', \\'ec5f9d01-21ea-46c1-b3da-8cbf24ce0eb7\\'\\n\\'x-envoy-expected-rq-timeout-ms\\', \\'15000\\'\""}
{"level":"debug","service":"envoy","name":"envoy","time":"2021-02-12T17:20:01Z"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:01Z","message":"[C8] using existing connection"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:01Z","message":"[C8] creating stream"}
{"level":"debug","service":"envoy","name":"router","time":"2021-02-12T17:20:01Z","message":"[C54][S18429224963929918799] pool ready"}
{"level":"debug","X-Forwarded-Proto":["https"],"ip":"127.0.0.1","user_agent":"kube-probe/1.16+","request-id":"ec5f9d01-21ea-46c1-b3da-8cbf24ce0eb7","duration":0.015206,"size":61,"status":200,"method":"GET","host":"10.52.3.67:443","path":"/ping","time":"2021-02-12T17:20:01Z","message":"http-request"}
{"level":"debug","service":"envoy","name":"router","time":"2021-02-12T17:20:01Z","message":"[C54][S18429224963929918799] upstream headers complete: end_stream=false"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:01Z","message":"coroutine finished"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:01Z","message":"coroutine finished"}
{"level":"debug","service":"envoy","name":"http","time":"2021-02-12T17:20:01Z","message":"\"[C54][S18429224963929918799] encoding headers via codec (end_stream=false):\\n\\':status\\', \\'200\\'\\n\\'content-type\\', \\'text/plain\\'\\n\\'date\\', \\'Fri, 12 Feb 2021 17:20:00 GMT\\'\\n\\'content-length\\', \\'61\\'\\n\\'x-envoy-upstream-service-time\\', \\'0\\'\\n\\'server\\', \\'envoy\\'\""}
{"level":"debug","service":"envoy","name":"envoy","time":"2021-02-12T17:20:01Z"}
{"level":"debug","service":"envoy","name":"client","time":"2021-02-12T17:20:01Z","message":"[C8] response complete"}
{"level":"debug","service":"envoy","name":"http2","time":"2021-02-12T17:20:01Z","message":"[C54] stream closed: 0"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:01Z","message":"[C8] response complete"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:01Z","message":"[C8] destroying stream: 0 remaining"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:01Z","message":"[C54] remote close"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:01Z","message":"[C54] closing socket: 0"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:01Z","message":"[C54] SSL shutdown: rc=1"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:01Z","message":"[C54] "}
{"level":"debug","service":"envoy","name":"conn_handler","time":"2021-02-12T17:20:01Z","message":"[C54] adding to cleanup list"}
{"level":"info","X-Forwarded-For":["172.26.52.70"],"X-Forwarded-Host":["authenticate.mycompany.com"],"X-Forwarded-Port":["443"],"X-Forwarded-Proto":["https"],"X-Real-Ip":["172.26.52.70"],"ip":"127.0.0.1","user_agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36","request-id":"3aba679b54f3d5a9d2c38dcf0b7f6f41","error":"Internal Server Error: authenticate.OAuthCallback: error redeeming authenticate code: identity/oidc: couldn't update user info identity/oidc: could not retrieve groups: Unauthorized","time":"2021-02-12T17:20:01Z","message":"httputil: ErrorResponse"}
{"level":"debug","service":"envoy","name":"router","time":"2021-02-12T17:20:01Z","message":"[C30][S8131583384802242911] upstream headers complete: end_stream=false"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:01Z","message":"coroutine finished"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:01Z","message":"coroutine finished"}
{"level":"debug","X-Forwarded-For":["172.26.52.70"],"X-Forwarded-Host":["authenticate.mycompany.com"],"X-Forwarded-Port":["443"],"X-Forwarded-Proto":["https"],"X-Real-Ip":["172.26.52.70"],"ip":"127.0.0.1","user_agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36","request-id":"3aba679b54f3d5a9d2c38dcf0b7f6f41","duration":2611.607247,"size":15554,"status":500,"method":"GET","host":"authenticate.mycompany.com","path":"/oauth2/callback?code=<redacted>&state=<redacted>","time":"2021-02-12T17:20:01Z","message":"http-request"}
{"level":"debug","service":"envoy","name":"http","time":"2021-02-12T17:20:01Z","message":"\"[C30][S8131583384802242911] encoding headers via codec (end_stream=false):\\n\\':status\\', \\'500\\'\\n\\'content-security-policy\\', \\'default-src \\'none\\'; style-src \\'self\\' data:; img-src * data:;\\'\\n\\'referrer-policy\\', \\'Same-origin\\'\\n\\'vary\\', \\'Cookie\\'\\n\\'x-pomerium-intercepted-response\\', \\'true\\'\\n\\'date\\', \\'Fri, 12 Feb 2021 17:20:00 GMT\\'\\n\\'content-type\\', \\'text/html; charset=utf-8\\'\\n\\'x-envoy-upstream-service-time\\', \\'2612\\'\\n\\'server\\', \\'envoy\\'\""}
{"level":"debug","service":"envoy","name":"envoy","time":"2021-02-12T17:20:01Z"}
{"level":"debug","service":"envoy","name":"client","time":"2021-02-12T17:20:01Z","message":"[C16] response complete"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:01Z","message":"[C16] response complete"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:01Z","message":"[C16] destroying stream: 0 remaining"}
{"level":"info","service":"envoy","upstream-cluster":"pomerium-control-plane-http","method":"GET","authority":"10.52.3.67:443","path":"/ping","user-agent":"kube-probe/1.16+","referer":"","forwarded-for":"","request-id":"ec5f9d01-21ea-46c1-b3da-8cbf24ce0eb7","duration":0.73027,"size":61,"response-code":200,"response-code-details":"via_upstream","time":"2021-02-12T17:20:01Z","message":"http-request"}
{"level":"info","service":"envoy","upstream-cluster":"pomerium-control-plane-http","method":"GET","authority":"authenticate.mycompany.com","path":"/oauth2/callback?code=<redacted>&state=<redacted>","user-agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36","referer":"","forwarded-for":"172.26.52.70","request-id":"3aba679b54f3d5a9d2c38dcf0b7f6f41","duration":2612.879987,"size":15554,"response-code":500,"response-code-details":"via_upstream","time":"2021-02-12T17:20:01Z","message":"http-request"}
{"level":"debug","service":"envoy","name":"main","time":"2021-02-12T17:20:03Z","message":"flushing stats"}
{"level":"debug","service":"envoy","name":"filter","time":"2021-02-12T17:20:03Z","message":"tls inspector: new connection accepted"}
{"level":"debug","service":"envoy","name":"conn_handler","time":"2021-02-12T17:20:03Z","message":"[C49] new connection"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:03Z","message":"[C49] handshake expecting read"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:03Z","message":"[C49] handshake expecting read"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:03Z","message":"[C49] handshake complete"}
{"level":"debug","service":"envoy","name":"http2","time":"2021-02-12T17:20:03Z","message":"[C49] updating connection-level initial window size to 268435456"}
{"level":"debug","service":"envoy","name":"http","time":"2021-02-12T17:20:03Z","message":"[C49] new stream"}
{"level":"debug","service":"envoy","name":"http","time":"2021-02-12T17:20:03Z","message":"\"[C49][S11452940698367822311] request headers complete (end_stream=true):\\n\\':authority\\', \\'10.52.3.64:443\\'\\n\\':method\\', \\'GET\\'\\n\\':path\\', \\'/ping\\'\\n\\':scheme\\', \\'https\\'\\n\\'user-agent\\', \\'kube-probe/1.16+\\'\\n\\'accept-encoding\\', \\'gzip\\'\""}
{"level":"debug","service":"envoy","name":"envoy","time":"2021-02-12T17:20:03Z"}
{"level":"debug","service":"envoy","name":"http","time":"2021-02-12T17:20:03Z","message":"[C49][S11452940698367822311] request end stream"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:03Z","message":"coroutine finished"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:03Z","message":"coroutine finished"}
{"level":"debug","service":"envoy","name":"router","time":"2021-02-12T17:20:03Z","message":"\"[C49][S11452940698367822311] cluster \\'pomerium-control-plane-http\\' match for URL \\'/ping\\'\""}
{"level":"debug","service":"envoy","name":"router","time":"2021-02-12T17:20:03Z","message":"\"[C49][S11452940698367822311] router decoding headers:\\n\\':authority\\', \\'10.52.3.64:443\\'\\n\\':method\\', \\'GET\\'\\n\\':path\\', \\'/ping\\'\\n\\':scheme\\', \\'http\\'\\n\\'user-agent\\', \\'kube-probe/1.16+\\'\\n\\'accept-encoding\\', \\'gzip\\'\\n\\'x-forwarded-proto\\', \\'https\\'\\n\\'x-request-id\\', \\'c0876f2a-97d9-43fa-b015-0cafca836d4c\\'\\n\\'x-envoy-expected-rq-timeout-ms\\', \\'15000\\'\""}
{"level":"debug","service":"envoy","name":"envoy","time":"2021-02-12T17:20:03Z"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:03Z","message":"[C5] using existing connection"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:03Z","message":"[C5] creating stream"}
{"level":"debug","service":"envoy","name":"router","time":"2021-02-12T17:20:03Z","message":"[C49][S11452940698367822311] pool ready"}
{"level":"debug","X-Forwarded-Proto":["https"],"ip":"127.0.0.1","user_agent":"kube-probe/1.16+","request-id":"c0876f2a-97d9-43fa-b015-0cafca836d4c","duration":0.020458,"size":61,"status":200,"method":"GET","host":"10.52.3.64:443","path":"/ping","time":"2021-02-12T17:20:03Z","message":"http-request"}
{"level":"debug","service":"envoy","name":"router","time":"2021-02-12T17:20:03Z","message":"[C49][S11452940698367822311] upstream headers complete: end_stream=false"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:03Z","message":"coroutine finished"}
{"level":"debug","service":"envoy","name":"lua","time":"2021-02-12T17:20:03Z","message":"coroutine finished"}
{"level":"debug","service":"envoy","name":"http","time":"2021-02-12T17:20:03Z","message":"\"[C49][S11452940698367822311] encoding headers via codec (end_stream=false):\\n\\':status\\', \\'200\\'\\n\\'content-type\\', \\'text/plain\\'\\n\\'date\\', \\'Fri, 12 Feb 2021 17:20:03 GMT\\'\\n\\'content-length\\', \\'61\\'\\n\\'x-envoy-upstream-service-time\\', \\'0\\'\\n\\'server\\', \\'envoy\\'\""}
{"level":"debug","service":"envoy","name":"envoy","time":"2021-02-12T17:20:03Z"}
{"level":"debug","service":"envoy","name":"client","time":"2021-02-12T17:20:03Z","message":"[C5] response complete"}
{"level":"debug","service":"envoy","name":"http2","time":"2021-02-12T17:20:03Z","message":"[C49] stream closed: 0"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:03Z","message":"[C5] response complete"}
{"level":"debug","service":"envoy","name":"pool","time":"2021-02-12T17:20:03Z","message":"[C5] destroying stream: 0 remaining"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:03Z","message":"[C49] remote close"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:03Z","message":"[C49] closing socket: 0"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:03Z","message":"[C49] SSL shutdown: rc=1"}
{"level":"debug","service":"envoy","name":"connection","time":"2021-02-12T17:20:03Z","message":"[C49] "}
{"level":"debug","service":"envoy","name":"conn_handler","time":"2021-02-12T17:20:03Z","message":"[C49] adding to cleanup list"}
{"level":"debug","service":"envoy","name":"main","time":"2021-02-12T17:20:03Z","message":"flushing stats"}
{"level":"info","service":"envoy","upstream-cluster":"pomerium-control-plane-http","method":"GET","authority":"10.52.3.64:443","path":"/ping","user-agent":"kube-probe/1.16+","referer":"","forwarded-for":"","request-id":"c0876f2a-97d9-43fa-b015-0cafca836d4c","duration":0.799989,"size":61,"response-code":200,"response-code-details":"via_upstream","time":"2021-02-12T17:20:03Z","message":"http-request"}
{"level":"debug","service":"envoy","name":"filter","time":"2021-02-12T17:20:03Z","message":"tls inspector: new connection accepted"}
{"level":"debug","service":"envoy","name":"filter","time":"2021-02-12T17:20:04Z","message":"tls inspector: new connection accepted"}
{"level":"debug","service":"envoy","name":"main","time":"2021-02-12T17:20:04Z","message":"flushing stats"}
{"level":"debug","service":"envoy","name":"main","time":"2021-02-12T17:20:05Z","message":"flushing stats"}
{"level":"debug","service":"envoy","name":"upstream","time":"2021-02-12T17:20:05Z","message":"transport socket match, socket default selected for host with address 172.25.54.188:80"}
{"level":"debug","service":"envoy","name":"upstream","time":"2021-02-12T17:20:05Z","message":"DNS refresh rate reset for prometheus-operator-grafana.monitoring.svc.cluster.local, refresh rate 30000 ms"}
{"level":"debug","service":"envoy","name":"upstream","time":"2021-02-12T17:20:05Z","message":"transport socket match, socket default selected for host with address 172.25.54.188:80"}
{"level":"debug","service":"envoy","name":"upstream","time":"2021-02-12T17:20:05Z","message":"DNS refresh rate reset for prometheus-operator-grafana.monitoring.svc.cluster.local, refresh rate 30000 ms"}
{"level":"debug","service":"envoy","name":"upstream","time":"2021-02-12T17:20:05Z","message":"transport socket match, socket default selected for host with address 172.25.54.229:9090"}
{"level":"debug","service":"envoy","name":"upstream","time":"2021-02-12T17:20:05Z","message":"DNS refresh rate reset for thanos-query.monitoring.svc.cluster.local, refresh rate 30000 ms"}
{"level":"debug","service":"envoy","name":"upstream","time":"2021-02-12T17:20:05Z","message":"transport socket match, socket default selected for host with address 172.25.54.229:9090"}
{"level":"debug","service":"envoy","name":"upstream","time":"2021-02-12T17:20:05Z","message":"DNS refresh rate reset for thanos-query.monitoring.svc.cluster.local, refresh rate 30000 ms"}
{"level":"debug","service":"envoy","name":"filter","time":"2021-02-12T17:20:05Z","message":"tls inspector: new connection accepted"}

```

**Comment by zackb** [comment]
I think if the Groups url was overridable, or if there's a way to disable the UserGroups function it would be a suitable workaround for the time being: https://github.com/pomerium/pomerium/blob/v0.9.5/internal/identity/oidc/onelogin/onelogin.go#L25

**Comment by calebdoxsey** [comment]
@zackb The version of Pomerium you are using is quite different from the latest version. We changed the way groups are queried. We now use a separate service account: https://www.pomerium.io/docs/identity-providers/one-login.html#service-account.

I was able to reproduce the issue about the wrong scopes, so I will make a PR to fix the default scopes, but I don't think we can fix the group issue you're running into, because we no longer query groups that way. We use the onelogin API instead: https://developers.onelogin.com/api-docs/1/groups/get-groups.

**Comment by zackb** [comment]
@calebdoxsey thanks for the quick reply and detailed explanation!

If you don't mind I have one more question. I looked to upgrade a while back and got stuck on the new data broker. As we have many independent k8s clusters and pomerium installations I couldn't get a databroker setup working across clusters. So a login to any of the pomerium instances worked across all of the rest. Do you happen to know of a way of accomplishing this that does not require cross-cluster communication? 

Thanks a ton

**Comment by calebdoxsey** [comment]
@zackb would it be possible for the various clusters to access to a shared redis instance? We have a redis backend for the databroker which should work with multiple databroker instances.

**Comment by zackb** [comment]
Hey @calebdoxsey unfortunately it's not an option for us for some pretty uninteresting reasons. I tried sharing a single in-memory cache between clusters and that works. But in the end I patched version 0.9.5 to parameterize the group url. This is a short term solution. Both of these actually work but ideally there'd be a cookie storage option again as that really fit our use case well, and the large cookie value didn't matter for this one. Thanks again for your help.
